### PR TITLE
open-cluster-management/backlog#13207 -- need information from dev

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: credentials/credential_ansible.adoc [hunk 1]
```diff
@@ -29,7 +29,7 @@ To create a credential from the {product-title} console, complete the following
 
 . Select *Add credential*.
 . Select *Red Hat Ansible Automation Platform* as your provider.
-. Add a name for your credential.
+. Add a name for your credential. // Also here, we need to tell user how to get this info
```
Comment (Randy424) [inline credentials/credential_ansible.adoc]: The name of the Ansible credential will come from the user.

Host - After authenticating to your Ansible Tower, go to Settings > System, see "BASE URL OF THE TOWER HOST"
 
Token - After authenticating to Tower > Users > select or create a user > Press "TOKEN" Tab to create a new authentication token.

You can find more information on creating a token [here ](https://docs.ansible.com/ansible-tower/latest/html/administration/oauth2_token_auth.html)

File: credentials/credential_ansible.adoc [hunk 2]
```diff
@@ -18,7 +18,7 @@ You must have the following prerequisites before creating a credential:
 * Internet access for your {product-title} hub cluster.
 * Ansible login credentials, which include access key ID and secret access key. 
 * Account permissions that allow installing clusters on Ansible. 
-// link here
+// link here-- where does user get this
```
Comment (Randy424) [inline credentials/credential_ansible.adoc]: To create Ansible tower credentials, the user needs "admin" privileges to create/view Ansible Tower credentials in their namespace.

Cluster creation using Ansible Job Templates will require a user role binding with role "open-cluster-management:cluster-manager-admin". 

*Not sure how to answer "where does user get this" (are you referring to the Ansible login creds, the hub cluster permissions, both?)

File: credentials/credential_ansible.adoc (hunk 1 above)
Comment (Randy424) [reply credentials/credential_ansible.adoc]: Information on the "BASE URL" w/r to Ansible Jobs, [here](https://docs.ansible.com/ansible-tower/3.4.0/html/administration/configure_tower_in_tower.html)

**Comment by Randy424** [comment]
@jnpacker Are there any other Ansible Docs you know of that we might want to link?

File: credentials/credential_ansible.adoc (hunk 2 above)
Comment (swopebe) [reply credentials/credential_ansible.adoc]: @Randy424 Sho said all credentials are Edit role capable-- I hear about three different answers so this is Edit for now and I may ask next week again.

**Comment by jnpacker** [comment]
https://docs.ansible.com/ansible-tower/

**Pull request merged**